cmd/kubeadm/app/features/features.go

```diff
@@ -23,21 +23,61 @@ import (
 	"strings"
 
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/util/version"
 )
 
 const (
+	// HighAvailability is alpha in v1.9
+	HighAvailability = "HighAvailability"
+
 	// SelfHosting is beta in v1.8
-	SelfHosting utilfeature.Feature = "SelfHosting"
+	SelfHosting = "SelfHosting"
 
 	// StoreCertsInSecrets is alpha in v1.8
-	StoreCertsInSecrets utilfeature.Feature = "StoreCertsInSecrets"
+	StoreCertsInSecrets = "StoreCertsInSecrets"
 )
 
+var v190 = version.MustParseSemantic("v1.9.0")
```

Comment:
_sigh_ that should probably be v1.9.0-alpha.1 :smile:

Reply:
@luxas Yep, I'll update this in a follow-up PR

Reply:
cool, thanks!